lua/entities/gmod_wire_expression2/core/number.lua

```diff
@@ -408,6 +431,10 @@ __e2setcost(2) -- approximation
 local deg2rad = pi / 180
 local rad2deg = 180 / pi
 
+registerFunction("huge", "", "n", function(self, args)
```

Comment:
It looks like pi was already a function before. If so, then keep it. It would be preferable to use a constant instead, but it's too late now. Removing it would break any old E2s using it.